apps/epic-web/src/styles/feedback-widget.css

```diff
@@ -85,3 +85,21 @@
 [data-sr-feedback-widget-category-field-buttons] button span {
   @apply after:pr-1 after:text-gray-600 after:content-['✓'] dark:after:text-gray-400;
 }
+
+/* seat selection field */
+
```

Comment:
components are going to be sharable so we can style them for each partner product